src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -555,8 +598,8 @@ Following the FSL format for the `[*_]dwi.bvec` specification, the coordinate sy
 the *b* vectors MUST be defined with respect to the coordinate system defined by
 the header of the corresponding `_dwi` NIfTI file and not the scanner's device
 coordinate system (see [Coordinate systems](../99-appendices/08-coordinate-systems.md)).
-The most relevant implication for this choice is that any rotations applied to the DWI data
```

Comment:
This is true for FSL format bvec files.

Reply:
This is true in general, I think.

Anyways, I agree this PR is not the right place for this suggestion. I'll roll this back and then bring the issue back to the table on the main PR where everyone can weigh in 👍